packages/mint-components/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@saasquatch/mint-components",
   "title": "Mint Components",
-  "version": "1.7.2",
+  "version": "1.7.3-2",
```

Comment:
Use stable version (`1.7.4` I think) for deploy

Reply:
looks like it'll be `1.7.5`